app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -8,6 +8,8 @@ class ManageIQ::Providers::Openshift::ContainerManager < ManageIQ::Providers::Co
   require_nested :RefreshWorker
   require_nested :Refresher
 
+  has_many :openshift_container_images, :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
No, it is just nice to have (I need to add ":class_name => ..." there though) do you want me to remove this?